megamek/src/megamek/common/Mounted.java

```diff
@@ -979,35 +978,60 @@ public void setHotLoad(boolean hotload) {
         }
 
     }
+    
+    /** Returns true when m is a PPC Capacitor and not destroyed. */
+    private boolean isWorkingCapacitor(Mounted m) {
+        return !m.isDestroyed()
+        && m.getType() instanceof MiscType
+        && ((MiscType) m.getType()).hasFlag(MiscType.F_PPC_CAPACITOR);
+    }
 
-    /**
-     * does this <code>Mounted</code> have a linked and charged PPC Capacitor?
+    /** 
+     * Returns 1 or 2 if this Mounted has a linked
+     * and charged (= set to charge in an earlier turn) 
+     * PPC Capacitor.
      */
-
     public int hasChargedCapacitor() {
+        if (getCrossLinkedBy() != null
+                && isWorkingCapacitor(getCrossLinkedBy())
+                && getCrossLinkedBy().curMode().equals("Charge")) {
+            return 2;
```

Comment:
Can we take these magic numbers and make them into constants?

Reply:
Or, even better, an enum?

Reply:
That is a problem of the original code. I only wanted to correct the error.